src/develop/tiling.c

```diff
@@ -1777,8 +1777,8 @@ static int _default_process_tiling_cl_roi(struct dt_iop_module_t *self, struct d
       piece->pipe->tiling = 1;
 
       /* the output dimensions of the good part of this specific tile */
-      const size_t wd = (tx + 1) * tile_wd > roi_out->width ? roi_out->width - tx * tile_wd : tile_wd;
```

Comment:
Why removing the `const` here ?

Maybe double check that above we can also add const ?

Reply:
copypasta got ahold of me here ;) will fix! thank!